bindings/cs/MongoDB.Libmongocrypt.Test32/MongoDB.Libmongocrypt.Test32.csproj

```diff
@@ -15,7 +15,7 @@
     </ItemGroup>
 
 
-    <ItemGroup Condition="'$(TargetFramework)'=='net452'">
+    <ItemGroup Condition="'$(TargetFramework)'=='net472'">
```

Comment:
`System.ValueTuple` was introduced in .NET Framework 4.7.0. So this shouldn't be required anymore.

https://docs.microsoft.com/en-us/dotnet/api/system.valuetuple